test/parallel/test-process-uncaught-exception-monitor.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const theErr = new Error('MyError');
+
+process.on(
+  process.uncaughtExceptionMonitor,
+  common.mustCall(function onUncaughtExceptionMonitor(err, origin) {

```

Comment:
Nit:

```suggestion
  common.mustCall((err, origin) => {
```

If applied, please also update the other functions to arrow functions.

Reply:
I tend to use named functions as it eases debugging a lot. Or is there some guideline to avoid named functions?

Reply:
We use named functions in case it's not e.g., a lambda function. Those normally use arrow functions. We also normally use arrow functions for mustCall / similar in our tests, since it's shorter and mostly intuitive what the function does.